commit f4ef2531698fb9ba006e8b31a223b3269be8bc7c
Author: Luis Chamberlain <mcgrof@kernel.org>
Date:   Wed Nov 21 17:07:45 2018 -0800

    README: add legacy notice
    
    As if kernel v4.15 CRDA is no longer needed. Annotate this. The
    code will still be maintained to help older kernels.
    
    Signed-off-by: Luis Chamberlain <mcgrof@kernel.org>

diff --git a/README b/README
index 88f2623..7d39891 100644
--- a/README
+++ b/README
@@ -3,11 +3,17 @@
 <https://wireless.wiki.kernel.org/en/developers/regulatory/crda>
 
 This is the Central Regulatory Domain Agent for Linux. It serves one
-purpose: tell Linux kernel what to enforce. In essence it is a udev
-helper for communication between the kernel and userspace. You only
-need to run this manually for debugging purposes. For manual changing
-of regulatory domains use iw (iw reg set) or wpa_supplicant (feature
-yet to be added).
+purpose: tell Linux kernel what regulatory ruels to enforce for 802.11.
+
+CRDA is no longer needed as of kernel v4.15 since commit 007f6c5e6eb45
+("cfg80211: support loading regulatory database as firmware file") added
+support to use the kernel's firmware request API which looks for the
+firmware on /lib/firmware. Because of this CRDA is legacy software for
+older kernels. It will continue to be maintained.
+
+CRDA is a udev helper for communication between the kernel and userspace. You
+only need to run this manually for debugging purposes. For manual changing of
+regulatory domains use iw (iw reg set) or wpa_supplicant.
 
  HOST REQUIREMENTS
 ===================
